scripts/delete-federation-with-helm-chart.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script removes the cluster registry, unjoins any clusters passed as
+# arguments, and removes the federation from the current kubectl context.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source "$(dirname "${BASH_SOURCE}")/util.sh"
+
+KCD="kubectl --ignore-not-found=true delete"
+NS="${FEDERATION_NAMESPACE:-federation-system}"
+PUBLIC_NS=kube-multicluster-public
+NAMESPACED="${NAMESPACED:-}"
+
+KF_NS_ARG="--federation-namespace=${NS} "
+if [[ "${NAMESPACED}" ]]; then
+  KF_NS_ARG+="--registry-namespace=${NS}"
```

Comment:
Why did `NAMESPACED` request `--registry-namespace`? Please check here https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/userguide.md#namespaced-federation for what does `NAMESPACED` mean.

Reply:
I did not touch this part. It follows the rule in delete-federation.sh. 

I can fix it if it is an issue.

Reply:
Yes. I checked related feature. It is used to test namespaced cluster registry.